angr/exploration_techniques/loop_seer.py

```diff
@@ -0,0 +1,140 @@
+import angr
+from . import ExplorationTechnique
+
+import logging
+from collections import defaultdict
```

Comment:
The order of import:

Common packages coming with Python
_blank line_
Third-party packages
_blank line_
**Relative import** from the current package